README.md

```diff
@@ -14,7 +14,18 @@ using the crates of the Rust ecosystem.
 These examples are complete, and suitable for copying directly into
 new cargo projects. They are tested and guaranteed to work.
 
+If you'd like to read it locally, [install Rust], and then:
```

Comment:
I'd suggest to put the description under separate heading like in the [rust-book readme](https://github.com/rust-lang/book/blob/master/README.md#building) although without the per browser instructions.

Also I see no reason to mention installing rust / rustup here, but we could hyperlink the rust lang mentions to the https://www.rust-lang.org/.